docs/concepts/accounts.md

```diff
@@ -0,0 +1,159 @@
+
+![Screenshot 2023-08-16 at 16.28.58.png](_accounts_images/Screenshot_2023-08-16_at_16.28.58.png)
+
+This represents a 2-of-3 multi-sig quorum, in which a transaction is authorized to access the account if it receives signatures from *at least* 2 out of 3 keys.
+
+An account key contains the following attributes:
+
+- **ID** used to identify keys within an account
+- **Public Key** raw public key (encoded as bytes)
+- **Signature algorithm** (see below)
+- **Hash algorithm** (see below)
+- **Weight** integer between 1-1000
+- **Revoked** whether the key has been revoked or it’s active
+- **Sequence Number** is a number that increases with each submitted transaction signed by this key
+
+### Signature and Hash Algorithms
+
+The signature and hashing algorithms are used during the transaction signing process and can be set to certain predefined values.
+
+There are two curves commonly used with the ECDSA algorithm, secp256r1 ([OID 1.2.840.10045.3.1.7](http://oid-info.com/get/1.2.840.10045.3.1.7), also called the "NIST P-256." curve), and secp256k1 ([OID 1.3.132.0.10](http://oid-info.com/get/1.3.132.0.10), the curve used by "Bitcoin"). Please be sure to double-check which parameters you are using before registering a key, as presenting a key using one of the curves under the code and format of the other will generate an error.
+
+| Algorithm | Curve     | ID              | Code |
+| --------- | --------- | --------------- | ---- |
+| ECDSA     | P-256     | ECDSA_P256      | 2    |
+| ECDSA     | secp256k1 | ECDSA_secp256k1 | 3    |
+
+*Please note that the codes listed here are for the signature algorithms as used by the node API, and they are different from the ones [defined in Cadence](../cadence/language/crypto.mdx#signing-algorithms)*
+
+| Algorithm | Output Size | ID       | Code |
+| --------- | ----------- | -------- | ---- |
+| SHA-2     | 256         | SHA2_256 | 1    |
+| SHA-3     | 256         | SHA3_256 | 3    |
+
+Both hashing and signature algorithms are compatible with each other, so you can freely choose from the set.
+
+### **Locked / Keyless Accounts**
+
+An account on Flow doesn’t require keys in order to exist, but this makes the account immutable since no transaction can be signed that would change the account. This can be useful if we want to freeze an account contract code and it elegantly solves the problem of having multiple account types (as that is the case for Ethereum).
+
+![Screenshot 2023-08-16 at 18.59.10.png](_accounts_images/Screenshot_2023-08-16_at_18.59.10.png)
+
+You can achieve keyless accounts by either removing an existing public key from an account signing with that same key and repeating that action until an account has no keys left, or you can create a new account that has no keys assigned.
+
+<Callout type="danger">
+Be careful when removing keys from an existing account, because once an account doesn’t have sufficient keys (keys with weights sum less than 1000), it can no longer be modified.
```

Comment:
```Be careful when removing keys from an existing account, because once an account doesn’t have sufficient keys (keys with weights sum less than 1000), it can no longer be modified.```

suggestion

```Be careful when removing keys from an existing account, because 1000 weight is necessary for any transactions on the account. If the account doesn’t have sufficient keys, it can no longer be modified.```

Reply:
included into https://github.com/onflow/docs/pull/211